crates/primitives-traits/Cargo.toml

```diff
@@ -20,7 +20,9 @@ alloy-genesis.workspace = true
 alloy-primitives.workspace = true
 alloy-rlp.workspace = true
 
-revm-primitives = { workspace = true, features = ["serde"] }
+# revm-primitives scroll re-export
+revm-primitives = { package = "reth-scroll-revm", path = "../scroll/revm", features = ["serde"] }
```

Comment:
makes sense yes, will udpate

Reply:
do you mean update `revm-primitives` to point to `reth-scroll-revm` from the workspace mainfest?

Reply:
I'm mean when we define `revm-primitives` in the workspace `Cargo.toml` we redefine it such that it points to `reth-scroll-revm`. Currently in workspace `Cargo.toml` we have:
```
revm-primitives = { version = "14.0.0", features = [
    "std",
], default-features = false }
```

We can change it to:
```
revm-primitives = { package = "reth-scroll-revm", path = "../scroll/revm", features = ["std"], default-features = false }
```

Then in this crate we shouldn't have to change the dependency.

Reply:
@frisitano I think this would work better in the follow up PR #39. The goal of the current PR was to introduce the Scroll crates with minimal propagation to the rest of the repo. Moving the dependency to workspace level would fit more 39's goal and limit the changes in this PR (which are done in 39 anyway).

Reply:
Yep sounds good.